plugins/synaptics-rmi/fu-synaptics-rmi-v7-device.c

```diff
@@ -941,7 +1077,10 @@ fu_synaptics_rmi_v7_device_setup(FuSynapticsRmiDevice *self, GError **error)
 	}
 
 	/* read flash config */
-	return fu_synaptics_rmi_v7_device_read_flash_config(self, error);
+	if (flash->bootloader_id[1] >= 10)
+		return TRUE;
+	else
+		return fu_synaptics_rmi_v7_device_read_flash_config(self, error);
```

Comment:
return, else -- if you push this branch to origin (e.g. blueue/synaptics-rmi) I can help you fix it up.